src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,38 +2,61 @@
 
 public class DefaultCountingOutRhymer {
 
+    private static final int TOTAL_FULL_VALUE = 11;
+
+    private static final int TOTAL_VALUE = -1;
+
+    private static final int MAX_SIZE = 12;
+
     // alt + <- powrót do wcześniej modyfikowanego pliku
     // alt + -> przejście do następnego pliku
-    private int[] numbers = new int[12];
 
-    public int total = -1;
+    private final int[] numbers = new int[MAX_SIZE];
 
-    public void countIn(int in) {
+    private int total = TOTAL_VALUE;
+
+    protected void countIn(int in) {
         if (!isFull()) {
             numbers[++total] = in;
         }
     }
 
-    public boolean callCheck() {
-        return total == -1;
+    protected boolean callCheck() {
+        return total == TOTAL_VALUE;
     }
 
-    public boolean isFull() {
-        return total == 11;
+    protected boolean isFull() {
+        return total == TOTAL_FULL_VALUE;
     }
 
     protected int peekaboo() {
         if (callCheck()) {
-            return -1;
+            return TOTAL_VALUE;

```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)